src/system_sensors.py

```diff
@@ -320,12 +394,36 @@ def send_config_message(mqttClient):
                 + f"\"unique_id\":\"{deviceName}_sensor_temperature\","
                 + f"\"availability_topic\":\"system-sensors/sensor/{deviceName}/availability\","
                 + f"\"device\":{{\"identifiers\":[\"{deviceName}_sensor\"],"
-                + f"\"name\":\"{deviceNameDisplay} Sensors\",\"model\":\"RPI {deviceNameDisplay}\", \"manufacturer\":\"RPI\"}},"
+                + f"\"name\":\"{deviceNameDisplay} Sensors\",\"model\":\"{deviceModel}\", \"manufacturer\":\"RPI Foundation\"}},"
```

Comment:
RPI Foundation if rasp in OS_DATA otherwise something else. Can we get the manufacture from the OS maybe?

Reply:
This would be a nice addition, I'll look into this.

Reply:
after a couple of searches and tries i came up with this :
`deviceManufacturer = "RPI Foundation" if "rasp" in OS_DATA["ID"] else OS_DATA['Name']`
this should give a sensible answer about the device/installation your running on.